src/pocketmine/command/defaults/BanCommand.php

```diff
@@ -49,9 +49,14 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 			return false;
 		}
 
-		$name = ctype_alnum(str_replace("_", "", array_shift($args)));
+		$name = str_replace("_", "", array_shift($args));
 		if(isset($args[0]) and isset($args[1])){
-			$reason = ctype_alnum(str_replace("_", "", $args[0]));
+			$reason = str_replace("_", "", $args[0]);
```

Comment:
There you can just replace this symbol. Or use another filter
`$reason = str_replace("|", "", $args[0]);`
The same must be used (there)[https://github.com/iTXTech/Genisys/blob/master/src/pocketmine/command/defaults/BanCommand.php#L63]